src/spec.c

```diff
@@ -1199,7 +1199,7 @@ void IndexSpec_GetStats(IndexSpec *sp, RSIndexStats *stats) {
   stats->numDocs = sp->stats.numDocuments;
   stats->numTerms = sp->stats.numTerms;
   stats->avgDocLen =
-      stats->numDocs ? (double)sp->stats.numRecords / (double)sp->stats.numDocuments : 0;
+      stats->numDocs ? (double)sp->stats.totalDocsLen / (double)sp->stats.numDocuments : 0;
```

Comment:
This is a bug fix also affecting the current non-standard BM25 scoring, right?
So this is a progression which is a breaking change?
Need to add to the PR a `breakingchange` label

Another question: If we would cherry-pick, we would take both the avg len bug fix together with the new BM25STD scorer, right?

Reply:
I don't see it as a breaking change, but a bug fix. It wouldn't affect the query results set, just the scores and perhaps the rankings.